Dockerfile

```diff
@@ -14,7 +14,11 @@ RUN apk -U add \
     ca-certificates \
     git \
     wget \
-    openssh-client && \
+    openssh-client \
+    python \
+    py-pip  && \
+    pip install --upgrade awscli && \
```

Comment:
Why need AWS CLI tool?

Reply:
@appleboy My bad. I committed on my master and forgot this PR was still open. I'll raise this change in a separate issue for discussion.